pandas/tests/io/parser/test_textreader.py

```diff
@@ -125,6 +129,45 @@ def test_integer_thousands_alt(self):
         expected = DataFrame([123456, 12500])
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "dtype",
+        [
+            "uint64",
+            "int64",
+            "uint32",
+            "int32",
+            "uint16",
+            "int16",
+            "uint8",
+            "int8",
+            "UInt64",
+            "Int64",
+            "UInt32",
+            "Int32",
+            "UInt16",
+            "Int16",
+            "UInt8",
+            "Int8",
+        ],
```

Comment:
can `any_int_dtype` fixture be used here?

Reply:
Absolutely, thanks. Should I also move this test into pandas/tests/io/parser/common/test_ints.py and check `pd.read_csv` for `all_parsers` (except pyarrow) due to my last comment? Just going for `TextReader` seems too specific now ...